extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -354,9 +402,35 @@ export class SystemLayer {
    */
   onMutedStateUpdate_(isMuted) {
     this.vsync_.mutate(() => {
-      isMuted ?
-        this.getShadowRoot().setAttribute(AUDIO_MUTED_ATTRIBUTE, '') :
-        this.getShadowRoot().removeAttribute(AUDIO_MUTED_ATTRIBUTE);
+      if (isMuted) {
+        this.getShadowRoot().setAttribute(AUDIO_MUTED_ATTRIBUTE, 'muted');
+      } else if (this.storeService_.get(StateProperty.PAGE_HAS_AUDIO_STATE)) {
+        this.getShadowRoot().setAttribute(AUDIO_MUTED_ATTRIBUTE, 'audioon');
```

Comment:
`muted="audioon"` doesn't read very well.  Maybe we should have this be another attribute?

I'd like us to leave the `muted` attribute the way it was so that it's in line with the way the `muted` attribute works on media elements: `muted` is present if the user has disallowed the element from making sound, otherwise it is not present.

Also, the `audioon` vs. `noaudio` vs. `muted` distinction is an internal one, so we should use an internal attribute or classname.  Maybe an attribute named `i-amphtml-story-audio-state`?